tests/jeos/firstrun.pm

```diff
@@ -21,7 +21,7 @@ use jeos qw(expect_mount_by_uuid);
 use utils qw(assert_screen_with_soft_timeout ensure_serialdev_permissions);
 use serial_terminal 'prepare_serial_console';
 
-my $user_created = 0;
+my $user_created = check_var('BACKEND', 'generalhw') ? 1 : 0;
```

Comment:
There is no obvious connection from `check_var('BACKEND', 'generalhw')` to "user was already created".